src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/view-your-opentelemetry-data-new-relic.mdx

```diff
@@ -61,38 +167,157 @@ If you send span events as described by the [OpenTelemetry specification](https:
+
+<table>
+  <thead>
+    <tr>
+      <th style={{ width: "200px" }}>
+        UI area
+      </th>
+      <th>
+        Attribute
+      </th>
+    </tr>
+  </thead>
+    <tbody>
+    <tr>
+      <td>
+        Top database calls
+      </td>
+      <td>
+        * `span.kind = client` or `producer`
+        * `db.system`
+        * Facets by span name (should follow DB semantic conventions - [TO_DO_FIND_LINK and CLARIFY span.name!])
+      </td>
+    </tr>
+    <tr>
+      <td>
+        Top databases (by query time)
+      </td>
+      <td>
+        * `span.kind = client` or `producer`
+        * `db.system`
+        * Facets by `db.system`
+  
+      </td>
+    </tr>
+    <tr>
+      <td>
+        Top databases (by throughput)
+      </td>
+      <td>
+        * `span.kind = client` or `producer`
+        * `db.system`
+        * Facets by `db.system`
+
+      </td>
+    </tr>
+    </tbody>
+</table>
+
+### Errors [#errors]
+
+On the Errors page, you can see total errors as well as charts showing error count and error rate.
+
+For your data to appear in this section, make sure it has the following:
+
+  * `span.kind = server` or `consumer`
+  * `otel.status_code = ERROR`
+  * Facets by span name
+
 ### Externals [#externals]
 
 Externals capture calls to out-of-process services such as web services, resources in the cloud, and other network entities.
 
+For your data to appear in this section, make sure it has the following:
+
+
+  * `span.kind = client` or `producer`
+  * `db.system` is not present
+
+### JVMs
+
+When you drill into a specific JVM, the UI display charts driven by JVM metric data:
+
+  * JVM metrics follow the general [semantic conventions for runtime environment metrics](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/runtime-environment-metrics.md#runtime-environment-specific-metrics---runtimeenvironment).
+  * The Java specific runtime metrics are not well documented. The [implementation](https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/runtime-metrics/library/src/main/java/io/opentelemetry/instrumentation/runtimemetrics) is effectively the documentation and may be subject to change. 
+
+For your data to appear in this section, make sure it has the following:
+
+  * Requires a unique `service.instance.id` attribute for rendering the list of JVMs
+  * `Service.instance.id` is an OpenTelemetry resource attribute
+
 ### Logs [#logs]
 
 The **Logs** page displays logs from your application. For more information about how to associate log data to your application in New Relic, see our [OpenTelemetry and logging documentation](/docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-logs).
 
+For your data to appear in this section, make sure it has the following:
+
+  * `service.name`
+  * To correlate with trace data, the logs should contain the `trace.id` and `span.id` attributes.
```

Comment:
improvement - to make this flow in the list, change to:

The `trace.id` and `span.id` attributes, to correlate with trade data

Reply:
@mmfred , good tip, but I'm going to hold off for the moment.